### ISSUE TITLE
Metrics Page Improvements & Documentation update

### ISSUE THREAD

**Issue by Kemi-Elizabeth** [platformsh/platformsh-docs#4256]
### Where on docs.platform.sh should be changed?

With the work to update and improve the Metrics page — the following pages will most likely require some changes:

https://docs.upsun.com/increase-observability/metrics.html

https://docs.platform.sh/increase-observability/metrics.html

https://docs.platform.sh/increase-observability/metrics/grid.html 

https://docs.platform.sh/increase-observability/metrics/dedicated.html 

https://docs.platform.sh/increase-observability/metrics/dedicated-generation-3.html

### What exactly should be updated?

Some changes required: Screenshots to reflect changes (e.g., now showing RAM for services on Platform.sh Grid, [removal of label mentions](https://docs.platform.sh/increase-observability/metrics.html#default-thresholds) ( critical, warning, burst   ) 

Changes are summarized in the following Slack thread:

https://platformsh.slack.com/archives/CPJNGJ84R/p1724786328964349?thread_ts=1720727602.738639&cid=CPJNGJ84R 

In short:

- Absolute values are changed to percentages on the Y-axis
- Threshold lines are still color-coded
- The threshold labels (warning, critical) are removed.
- These changes apply across all infrastructures and on Upsun/Platform.sh 
- Also, the RAM graph for services is back on Platform.sh — It was removed a year ago but is brought back along with the changes above to remove anxiety. 

**Design Environment with changes:** https://design-review-4-sv7thiy-i7dymnob47qsi.us-2.platformsh.site/pixelart/2jb5aslbftb24/master/metrics?range=15m
